src/config/project-statuses.js

```diff
@@ -0,0 +1,6 @@
+export default Object.freeze({
+  NEW: "new",
```

Comment:
I'm not sure this is really useful. It's a full new file, far from where it will be used, will be used at only one place, doing very little.
Declaring it in utils/projects.js makes more sense IMHO, or even don't use it.